pages/landing/index.js

```diff
@@ -74,6 +82,58 @@ export const getStaticProps = async () => {
           img: '/images/placeholder.jpg',
         },
       ],
+      stats: [
+        {
+          isPercentage: true,
+          number: 99,
+          description:
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pretium bibendum mauris, mollis...',
+        },
+        {
+          isPercentage: false,
+          number: 12,
+          description:
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pretium bibendum mauris, mollis...',
+        },
+        {
+          isPercentage: true,
```

Comment:
is `isPercentage` necessary? Why cant we just put `99%` as text? I dont think all stats are just numbers or percentages. what if we want `100K` or `9/10 students...` This will probably change the prop types.

Reply:
Good point. I can dumb it down.